sympy/polys/matrices/sdm.py

```diff
@@ -45,14 +82,82 @@ def __repr__(self):
 
     @classmethod
     def new(cls, sdm, shape, domain):
+        """
+
+        Parameters
+        ==========
+        sdm: A dict of dicts for non-zero elements in SDM
+        shape: Represents dimension of SDM
+        domain: Represents :py:class:`~.Domain` of SDM
```

Comment:
There should be a blank line after `=====`.

The description of the parameters should actually say what the are e.g. `shape` is a tuple.